contracts/MedianOracle.sol

```diff
@@ -105,14 +112,22 @@ contract MedianOracle is Ownable, IOracle {
         reports[index_past].payload = payload;
     }
 
+    /**
+    * @notice Invalidates the reports of the calling provider.
+    */
     function purgeReports() external
     {
         address providerAddress = msg.sender;
         require (providerReports[providerAddress][0].timestamp > 0);
         providerReports[providerAddress][0].timestamp=1;
         providerReports[providerAddress][1].timestamp=1;
     }
-
+    /**
+    * @notice Computes median of provider reports that their timestamps are in
```

Comment:
Computes median of provider reports whose timestamps are in the valid timestamp range.